src/components/gui/gui.jsx

```diff
@@ -12,6 +12,16 @@ const Box = require('../box/box.jsx');
 const Tab = require('../tab/tab.jsx');
 const TabText = require('../tab/tabText.jsx');
 
+function tabOne {
+    this.setState({tabOne: true, tabTwo: false, tabThree: false});
+}
+function tabTwo {
+    this.setState({tabOne: false, tabTwo: true, tabThree: false});
+}
+function tabThree {
+    this.setState({tabOne: false, tabTwo: false, tabThree: true});
+}
```

Comment:
This is a syntax error (should be `function foo() {`, not `function foo {`).

You should test your code on your own computer before you make a pull request.. :P